core/modules/widgets/vars.js

```diff
@@ -0,0 +1,79 @@
+/*\
+title: $:/core/modules/widgets/vars.js
+type: application/javascript
+module-type: widget
+
+This widget allows the user to set multiple variables in one go.
+It thereby reduces code complexity that would arise when setting
```

Comment:
I think you can cut this second sentence; this isn't the place to sell the advantages of the widget - you've already successfully sold it :)